README.md

```diff
@@ -1,11 +1,31 @@
 # STAC ipyleaflet
 
+### What
+stac_ipyleaflet is a customized version of [ipyleaflet](https://ipyleaflet.readthedocs.io/en/latest/) built to be an in jupyter-notebook interactive mapping library that can access STAC catalogs (currently only the MAAP STAC) to help better visualize cloud-optimized geotiffs. The library provides widgets that is meant to help users working in Jupyter Notebook environments to quickly visualize and interact with geospatial data at different stages throughout their process.
```

Comment:
Just a few suggested edits, to generalize away from some of the currently existing limitations:

`stac_ipyleaflet` is a customized version of [`ipyleaflet`](https://ipyleaflet.readthedocs.io/en/latest/) built to be an in-jupyter-notebook interactive mapping library that prioritizes access to STAC catalog data. The library provides ([`ipywidgets`](https://ipywidgets.readthedocs.io/en/stable/)-based) components that are meant to help users working in Jupyter Notebook environments to quickly visualize and interact with geospatial data at different stages throughout their research process.